packages/lib/convertToNewDurationType.ts

```diff
@@ -0,0 +1,35 @@
+export const MINUTES_IN_HOUR = 60;
+export const MINUTES_IN_DAY = 1440;
+export const HOURS_IN_DAY = 24;
+
+export default function convertToNewDurationType(prevType: string, newType: string, prevValue: number) {
+  if (newType === "minutes") {
+    if (prevType === "hours") {
+      console.log("hours to minutes", prevValue * MINUTES_IN_HOUR);
```

Comment:
Could you please remove this logs again? 😄

Reply:
@JeroenReumkens Oh right! I'm so sorry about that! I completely forgot about them! Thank you so much for reminding me!